drivers/volume/portworx/portworx.go

```diff
@@ -276,6 +285,30 @@ func (d *portworx) InspectVolume(name string, params map[string]string) error {
 	return nil
 }
 
+func (d *portworx) ValidateDeleteVolume(vol *vol.Volume) error {
+	name := d.schedOps.GetVolumeName(vol)
+	t := func() (interface{}, error) {
+		vols, err := d.volDriver.Inspect([]string{name})
```

Comment:
I think volDriver.Inspect will return openstorage's ErrEnoEnt error if it does not find a volume. You can add a specific check if ErrEnoEnt then return success